ui/raidboss/data/06-ew/trial/zodiark-ex.ts

```diff
@@ -1,14 +1,119 @@
 import NetRegexes from '../../../../../resources/netregexes';
+import { UnreachableCode } from '../../../../../resources/not_reached';
+import Outputs from '../../../../../resources/outputs';
+import { callOverlayHandler } from '../../../../../resources/overlay_plugin_api';
 import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
+import { PluginCombatantState } from '../../../../../types/event';
 import { TriggerSet } from '../../../../../types/trigger';
 
-export type Data = RaidbossData;
+export interface Data extends RaidbossData {
+  sigilMap: { [id: string]: 'Blue' | 'Red' };
+  storedSigils: { x: number; y: number; id: string }[];
+  storedTetherTargets: { [id: string]: PluginCombatantState };
+  storedPythons: PluginCombatantState[];
+  // For Quetz, mechanics which only use two of them always use the two with the highest ID
+  storedQuetzalcoatls: PluginCombatantState[];
+  storedBehemoths: PluginCombatantState[];
+  exoterikosCounter: number;
+  paradeigmaCounter: number;
+  astralFlowCounter: number;
+  adikiaCounter: number;
+}
+
+const directionOutputStrings = {
+  northeast: Outputs.northeast,
+  north: Outputs.north,
+  northwest: Outputs.northwest,
+  west: Outputs.west,
+  southeast: Outputs.southeast,
+  south: Outputs.south,
+  southwest: Outputs.southwest,
+  east: Outputs.east,
+  combo: {
+    en: '${dir1} / ${dir2}',
+  },
+};
+
+const pythonLineOutputStrings = {
+  '0': Outputs.num1,
+  '1': Outputs.num2,
+  '2': Outputs.num3,
+  '3': Outputs.num4,
+  'row': {
+    en: 'Row ${first}/${second}',
```

Comment:
I'm a little bit skeptical of Row/Column outputs being something that will be meaningful to a lot of people.

What do you think about some form like one of these:
* North all the way
* Slightly north of center
* Just east of center
* All the way west

For the later paradeigma with snakes, then maybe something like "NE (lean S)", since folks will already be constrained to a small area.